include/flamegpu/visualiser/PanelVis.h

```diff
@@ -0,0 +1,291 @@
+#ifndef INCLUDE_FLAMEGPU_VISUALISER_PANELVIS_H_
+#define INCLUDE_FLAMEGPU_VISUALISER_PANELVIS_H_
+#include <memory>
+#include <string>
+#include <utility>
+#include <unordered_map>
+#include <set>
+
+#include "flamegpu/model/EnvironmentDescription.h"
+#include "flamegpu/visualiser/config/ModelConfig.h"
+
+namespace flamegpu {
+namespace visualiser {
+
+/**
+ * This class serves as an interface for managing an instance of PanelConfig
+ * It allows elements to be specified for a UI panel for the visualisation
+ */
+class PanelVis {
```

Comment:
Not super keen on the class name (don't like `Vis`, but it matches exisitng use so not going to fight it). 

`VisPanel` would make more snse though? otherwise given the existance of `ModelVis` it implies its currently visualising a panel? Feel completely free to ignor this.